components/taxonomy/tests/test_manager.rs

```diff
@@ -26,12 +26,12 @@ fn target_map<K, T>(mut source: Vec<(Vec<K>, T)>) -> TargetMap<K, T> where K: Cl
     source.drain(..).map(|(v, t)| Targetted::new(v, t)).collect()
 }
 
-trait Transform<T> {
-    fn transform(&self) -> T;
+trait AsValue {
+    fn as_value(&self) -> Value;
 }
 
-impl Transform<Value> for (Payload, Arc<Format>) {
-    fn transform(&self) -> Value {
+impl AsValue for (Payload, Arc<Format>) {
+    fn as_value(&self) -> Value {
         self.0.to_value(&self.1).unwrap()
     }
 }
```

Comment:
although it's not totally the same as this would take the ownership of self. OK forget this comment please :)